conda_libmamba_solver/shards.py

```diff
@@ -438,6 +456,8 @@ def batch_retrieve_from_cache(sharded: list[Shards], packages: list[str]):
     sharded = [shardlike for shardlike in sharded if isinstance(shardlike, Shards)]
 
     wanted = []
+    # XXX update batch_retrieve_from_cache to work with (Shards, package name)
+    # tuples instead of broadcasting across shards itself.
```

Comment:
batch_retrieve_from_cache will change in the multithreaded refactor, no reason to change it here